main.tf

```diff
@@ -102,6 +102,11 @@ resource "kubernetes_deployment" "wandb" {
             value = var.redis_connection_string
           }
 
+          env {
+            name  = "GORILLA_CUSTOM_METRICS_PROVIDER"
```

Comment:
Is GORILLA (an internal name) something that should be user-facing? Should we consider renaming this?